applications/doodle/src/module/cf_sms_user.erl

```diff
@@ -25,7 +25,10 @@
 -spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
 handle(Data, Call1) ->
     UserId = wh_json:get_ne_value(<<"id">>, Data),
-    Call = doodle_util:set_callee_id(UserId, Call1),
+    Funs = [{fun doodle_util:set_callee_id/2, UserId}
```

Comment:
@aberke the callee is being set since Dec 22, 2014 https://github.com/2600hz/kazoo/commit/e40097b03174cd61e298202a987eed56847f24e4